spec/services/encryption/contextless_kms_client_spec.rb

```diff
@@ -148,13 +149,16 @@
       end
 
       it 'logs the encryption' do
-        expect(Encryption::KmsLogger).to receive(:log).with(
-          :encrypt,
-          log_context: { context: 'abc' },
-          key_id: IdentityConfig.store.aws_kms_key_id,
-        )
+        Timecop.freeze(log_timestamp) do
+          expect(Encryption::KmsLogger).to receive(:log).with(
+            action: :encrypt,
+            timestamp: log_timestamp,
+            log_context: { context: 'abc' },
+            key_id: IdentityConfig.store.aws_kms_key_id,
+          )
```

Comment:
```suggestion
        expect(Encryption::KmsLogger).to receive(:log).with(
          action: :encrypt,
          timestamp: log_timestamp,
          log_context: { context: 'abc' },
          key_id: IdentityConfig.store.aws_kms_key_id,
        )
```

With specifying a timestamp, I think freezing time shouldn't be necessary. Current preference is also to use Rails' internal [TimeHelpers](https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html).